src/HtmlBuilder.php

```diff
@@ -461,7 +449,7 @@ protected function attributeElement($key, $value)
         }
 
         if (! is_null($value)) {
-            return $key . '="' . $this->escapeAll($value) . '"';
+            return $key . '="' . e($value) . '"';
         }
```

Comment:
Disagree for attributes like `value`, because they're literal, not for display. Unfortunately this function can't make that distinction, because it doesn't know context.